### ISSUE TITLE
Bankers' Rounding - Non-Bankers' Rounding

### ISSUE THREAD

**Issue by mburns08109** [twinbasic/twinbasic#1071]
**Is your feature request related to a problem? Please describe.**
I see that tB echo's VBx's Bankers rounding. Fine and good. but can we have a NON-Bankers' rounding alternative function too?  (assuming you're not already ahead of me on this, Wayne).

**Describe the solution you'd like**
a RealRound(nm precision) (or RRound(), for short if you like) function, perhaps?

**Describe alternatives you've considered**
same-old, same-old

**Additional context**
It'd just be nice to have an arithmetic Round function just as handy as the bankers rounding is already.
(I've long thought that rounding should be controlled by a module-level OPTION command
OPTION BANKERS  or  OPTION ARITMETIC with OPTION BANKERS being the default for VBx back-compat)

**Comment by alberto-gilman** [comment]
it coud be done adding to Round Function one optional parameter Banker = True

**Comment by bclothier** [comment]
Keep in mind that there are [several rounding methods](https://en.wikipedia.org/wiki/Rounding). If one wants to have native implementation of those methods, an enum would do the job better even if we only enable a subset of the different methods available.

**Comment by mburns08109** [comment]
@bclothier Yeah, in the end I don't care if it's done by separate functions, OPTION settings, or adding (optional) enum parameters to the existing rounding functions, so long as there is _some_ documented, supported way to override the default rounding algorithm.

**Comment by bclothier** [comment]
For the record I don't think a `Option` statement is appropriate. The more local, the better, IMO.

**Comment by mburns08109** [comment]
@bclothier Oh, i don't know...a module-level OPTION could reset the default for all calculations in all module code.
On the other hand, it could lead to subtle bugs when calls go across module boundaries, and the rounding is different...

Ok, you've convinced me...the more local (and explicit) the better.
